lib/http-api.js

```diff
@@ -205,8 +205,8 @@ function parseText(str) { return str; }
  * Detect session expiry
  * @protected
  */
-HttpApi.prototype.isSessionExpired = function(response) {
-  return response.statusCode === 401;
+HttpApi.prototype.isSessionExpired = function(response, request) {
+  return response.statusCode === 401 || (response.statusCode === 403 && request.url.indexOf('login') > -1);
```

Comment:
Cannot be determined it is login server URL (identity URL) even it includes a "login" word